config/main.json

```diff
@@ -3,7 +3,7 @@
   "githubUrl": "https://github.com",
   "pullRequests": {
     "allowedActions": ["opened", "reopened", "synchronize"],
-    "allowedExtensions": [".js", ".json"],
+    "allowedExtensions": [".js", ".json", ".yaml"],
```

Comment:
Let's add `.yml` as well. I will plan to get rid of this config line as it should get a list of allowed extensions **automatically** from configured filters (`config/filters.json`) but it needs to be here for now.

Reply:
added